### PR TITLE
[docs]: translate hertz tutorials/basic-feature/route/ 

### PR DESCRIPTION
**Pull request by justlorain**
#### What type of PR is this?

docs: Documentation only changes

#### What this PR does / why we need it (en: English/zh: Chinese):

en: Added English documentation for hertz tutorials/basic-feature/route/ (route.md)
zh: 增加了hertz tutorials/basic-feature/route/部分的英文文档（route.md）

#### Which issue(s) this PR fixes:

[docs]: translate hertz tutorials/basic-feature/route/ 
https://github.com/cloudwego/hertz/issues/233

### CODE REVIEW

**Comment by GuangmingLuo** [comment]
Please update the index of the two others files : https://github.com/cloudwego/cloudwego.github.io/pull/334

**Comment by justlorain** [comment]
Sorry that I'm not sure about how to update the index of the other files. Could you please explain it more clearly, thx.

File: content/en/docs/hertz/tutorials/basic-feature/route.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	"context"
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
+)
+
+func main(){
+	h := server.Default(server.WithHostPorts("127.0.0.1:8080"))
+	v1 := h.Group("/v1")
+	h.GET("/get", func(ctx context.Context, c *app.RequestContext) {
```
Comment (li-jin-gou) [inline content/en/docs/hertz/tutorials/basic-feature/route.md:71]: there is a problem with the code here

**Comment by GuangmingLuo** [comment]
> Sorry that I'm not sure about how to update the index of the two other files. Could you please explain it more clearly, thx.

@justlorain hi，check this pr, and you will know how to do it : https://github.com/cloudwego/cloudwego.github.io/pull/334

File: content/en/docs/hertz/tutorials/basic-feature/route.md (hunk 1 above)
Comment (GuangmingLuo) [reply content/en/docs/hertz/tutorials/basic-feature/route.md:71]: https://github.com/cloudwego/cloudwego.github.io/pull/348

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**